src/control/SConscript

```diff
@@ -147,9 +147,18 @@ def scons():
     gosrc = Dir('.').srcnode().abspath
     cartpath = Dir('#/src/cart/src/include').srcnode().abspath
 
-    denv.AppendENVPath("CGO_LDFLAGS",
-                       denv.subst("-L$BUILD_DIR/src/cart/src/gurt"), sep=" ")
-    denv.AppendENVPath("CGO_CFLAGS", "-I%s" % cartpath, sep=" ")
+    prereqs.require(denv, 'ofi', 'hwloc')
+
+    cgolibdirs = denv.subst("-L$BUILD_DIR/src/cart/src/gurt "
+                            "-L$PREFIX/lib64 "
+                            "-L$HWLOC_PREFIX/lib -L$OFI_PREFIX/lib "
+                            "$_RPATH")
+    denv.AppendENVPath("CGO_LDFLAGS", cgolibdirs, sep=" ")
+
+    denv.AppendENVPath("CGO_CFLAGS", "-I%s " % cartpath +
+                       denv.subst("-I$HWLOC_PREFIX/include "
+                                  "-I$OFI_PREFIX/include"),
```

Comment:
Same comment about hwloc.   We should also consider using $_CCFLAGS

Reply:
Removed references to HWLOC_PREFIX